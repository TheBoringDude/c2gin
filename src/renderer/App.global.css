@tailwind base;
@tailwind components;
@tailwind utilities;

body {
  @apply bg-white dark:bg-warmGray-800;
}

.list-scroll {
  overflow-y: scroll;
  scrollbar-width: none; /* Firefox */
  -ms-overflow-style: none; /* Internet Explorer 10+ */
}
.list-scroll::-webkit-scrollbar {
  /* WebKit */
  width: 0;
  height: 0;
}
