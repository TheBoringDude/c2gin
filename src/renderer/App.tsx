import React from 'react';
import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';

// start import all font css (doesn't work if importing just @fontsource/inter)
import '@fontsource/catamaran/300.css';
import '@fontsource/catamaran/400.css';
import '@fontsource/catamaran/700.css';
import '@fontsource/catamaran/900.css';
// end import all font css (doesn't work if importing just @fontsource/inter)

import './App.global.css';

import Home from './pages/home';

export default function App() {
  return (
    <main className="w-11/12 mx-auto">
      <Router>
        <Switch>
          <Route path="/">
            <Home />
          </Route>
        </Switch>
      </Router>
    </main>
  );
}
