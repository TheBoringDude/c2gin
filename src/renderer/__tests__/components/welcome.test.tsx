import '@testing-library/jest-dom';
import React from 'react';
import renderer from 'react-test-renderer';
import WelcomeMessage from '../../components/welcome';

describe('Welcome Message', () => {
  it('should render', () => {
    const wrapper = renderer.create(<WelcomeMessage />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
