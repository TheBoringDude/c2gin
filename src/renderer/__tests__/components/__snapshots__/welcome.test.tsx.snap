// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome Message should render 1`] = `
<div
  className="p-8 text-center"
>
  <div
    className="w-5/6 mx-auto"
  >
    <h3
      className="text-4xl font-bold text-gray-600 dark:text-gray-200"
    >
      Welcome back!
    </h3>
    <div
      className="my-12"
    >
      <div
        className="p-6 border rounded-lg my-4 dark:border-gray-700 bg-white dark:bg-warmGray-900"
      >
        <img
          alt="c2gin - Manage Projects"
          className="h-56 md:h-64 mx-auto"
          src="test-file-stub"
        />
        <p
          className="text-xl text-gray-600 py-4 dark:text-gray-200"
        >
          Manage the things needed to do in your project
        </p>
      </div>
    </div>
    <div
      className="mt-20 flex items-center justify-between"
    >
      <a
        className="text-gray-600 dark:text-gray-200 hover:text-indigo-600 hover:underline"
        href="https://github.com/TheBoringDude/c2gin"
        rel="noreferrer"
        target="_blank"
        title="Goto Github Repo"
      >
        Github
      </a>
      <a
        className="text-gray-600 dark:text-gray-200 hover:text-indigo-600 hover:underline"
        href="https://github.com/TheBoringDude"
        rel="noreferrer"
        target="_blank"
        title="Author"
      >
        @TheBoringDude
      </a>
    </div>
  </div>
</div>
`;
